src/screens/LegalRights/VideoPage/index.tsx

```diff
@@ -0,0 +1,28 @@
+import { Pressable, ScrollView, Text, View } from 'react-native';
+import { Video } from 'expo-av';
+import { styles } from './styles';
+
+export default function VideoPage() {
+  return (
+    <ScrollView style={styles.container}>
+      <Video
+        source={require('@/assets/videos/da_link.mp4')}
+        rate={1.0}
+        volume={10.0}
```

Comment:
nitpick but just leave this at 1, in the docs it says volume is an integer between 0.0 and 1.0. So just set it to 1 for now